javaslang/src/main/java/javaslang/collection/BitMappedTrie.java

```diff
@@ -191,39 +192,6 @@ T get(int index) {
         }
     }
 
-    Iterator<T> iterator() {
```

Comment:
since we have `trailing` and `leading` arrays, we move the iterator to include those also